### DISCUSSION TITLE
Is it possible to process Pylance's "quick fix" in batches, or to process the same type together?

### DISCUSSION THREAD

**Discussion by Jill-Cheng** [microsoft/pylance-release#2667] [Enhancement]
When using the extension Pylance in VSCode, it will display errors or warnings in the code in the terminal. And we can use its "quick fix" to solve the problems in the code, but it is fixed one by one. 

Is it possible for it to be processed in batches at the same time, or the same type can be processed together?

For example: (It is repaired one by one , and when there are many "quick fixes", the workload will be heavy.)

![1](https://user-images.githubusercontent.com/61971386/102851865-d6b59500-4457-11eb-9ee9-70e5219a3706.gif)

![image](https://user-images.githubusercontent.com/61971386/102851121-f3e96400-4455-11eb-8e8b-d0d7d62e5627.png)

Related stack overflow link: [Visual code + Python: how to quick fix all?](https://stackoverflow.com/questions/65395204/visual-code-python-how-to-quick-fix-all)

Thanks a lot!

**Comment by PabloLION** [comment]
I see this issue is created more than one year. Are we going to fix it?

**Comment by erictraut** [comment]
We generally prioritize feature requests based on interest level. By expressing your interest in this feature (e.g. by adding a thumbs up), it will increase the likelihood that it will be considered at some point. We appreciate your input.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
